micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -258,11 +257,6 @@ public Search tags(String... tags) {
             return tags(zip(tags));
         }
 
-        public Search value(Statistic statistic, double value) {
```

Comment:
I removed `value()` since none of our tests are using it anymore. I can add it back if you think it belongs in the public API.

Reply:
Yep, it's good to put this one to bed.